ui/pages/confirmation/confirmation.js

```diff
@@ -32,11 +32,13 @@ import {
   ///: END:ONLY_INCLUDE_IN
   getUnapprovedTemplatedConfirmations,
   getUnapprovedTxCount,
+  hasPendingApprovalFlows as hasPendingApprovalFlowsSelector,
```

Comment:
Should we append the word `Selector` at the end of our boolean selector methods that start with `has`? Otherwise we are going to find this problem very often.

Another options would be not having selectors that are not prefaced by the word `get` or `select`, which seems to be the naming convention as far as I can see (https://redux.js.org/style-guide/#name-selector-functions-as-selectthing).

That'd mean having to check the length every time to get this behaviour.